feat: Add Testimonials Slider slice

dinartecc opened the pull request:
The original Vue proyect displayed the images at a set size of 150x150px, disregarding the `dimensions`  property inside the mock object. It may need to be changed so that it takes the size that is sent through the dimensions property.

dinartecc committed:
Add Testimonials Slider slice

dinartecc committed:
Fix Slider component stylesheet

The container for the slider dots is 40px tall. Fixed .slick-dots bottom property
from -25px to -40px to properly display below the main container.

georch commented on src/slices/TestimonialsSlider/README.md:
```diff
@@ -0,0 +1,24 @@
+# Pricing Table Section
```
```suggestion
# Testimonials Slider Section
```

georch commented on src/slices/TestimonialsSlider/model.json:
```diff
@@ -0,0 +1,75 @@
+      }
+    }
+  }
+}
```
```suggestion
}

```

georch commented on src/slices/TestimonialsSlider/index.js:
```diff
@@ -0,0 +1,98 @@
+import { Box } from 'theme-ui'
+import { RichText } from 'prismic-reactjs'
+import uuid from 'react-uuid'
+
```
```suggestion

import { structuredTextPropTypes } from '../../utils/prop-types'
```

georch commented on src/slices/TestimonialsSlider/index.js:
```diff
@@ -0,0 +1,98 @@
+TestimonialsSlider.propTypes = {
+  slice: PropTypes.shape({
+    primary: PropTypes.shape({
+      eyebrow_headline: PropTypes.array,
+      title: PropTypes.array,
+      description: PropTypes.array,
```
```suggestion
      eyebrow_headline: structuredTextPropTypes,
      title: structuredTextPropTypes,
      description: structuredTextPropTypes,
```

dinartecc committed:
Add custom dot label functionality to Slider component

dinartecc committed:
Fix suggested changes in Testimonials Slider slice

- Proper title in README.md
- Custom dot label in index.js
- Proper propTypes in index.js
- Blank line at EOF in model.json
